dev-utils/test-config.js

```diff
@@ -40,7 +40,7 @@ function getTestEnvironmentVariables() {
     branch: process.env.BRANCH_NAME,
     mode: process.env.MODE,
     sauceLabs: process.env.MODE && process.env.MODE.startsWith('saucelabs'),
-    isJenkins: process.env.JENKINS_URL,
+    isCI: process.env.JENKINS_URL || process.env.GITHUB_API_URL,
```

Comment:
We could also use the `CI` environment variable. It's supported in both Jenkins and GitHub workflow. Wdyt?